client/src/pages/Demo/Demo.js

```diff
@@ -1,25 +1,11 @@
 import React from 'react';
-import { Container } from 'react-bootstrap';
-import { ProgressCircle } from '../../components/progress-circle/ProgressCircle';
-import { Color } from '../../helpers/enums/Colors';
-import { ProductTable } from '../../components/product-table/ProductTable';
+import {Container} from 'react-bootstrap';
 
 export const DemoPage = () => {
+
```

Comment:
Tutaj tej zawrtości demoPage nie wyrzucaj, mozemy to potraktowac jako dokumnetacje komponentow